libraries/helper.rb

```diff
@@ -74,9 +74,9 @@ def windows_cleanpath(path)
       end
 
       def application_cleanname(application_name)
-        if !application_name.include?('/') && !application_name.end_with?('/')
+        if application_name.count('/') == 0
```

Comment:
same as previous but is more clear, if the item doesn't include a '/' and doesn't end in one (weird statement since if it doesn't contain one it likely doesn't end in one....
return the string with a '/' on the end.